src/test/java/InvalidLoginTests.java

```diff
@@ -0,0 +1,25 @@
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LoginPage;
+
+public class InvalidLoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage = new LoginPage(driver);
+        driver.get(url);
+    }
+    @Test
+    public void invalidPasswordEmail(){
+        loginPage.provideEmail("incorrectEmail").providePassword("incotrerctPwd").clickSubmit();
+    }
+    @Test
+    public void emptyEmail(){
+        loginPage.provideEmail("").providePassword("incotrerctPwd").clickSubmit();
+    }
+    @Test
+    public void emptyEmailPassword(){
+        loginPage.provideEmail("").providePassword("").clickSubmit();
+    }
+}
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/LiudmilaChernyh/A49-QA-Automation-Framework/assets/104761482/4618b855-6c93-4574-b5a4-64ecfabb5bce)